cardio_crumble.opam

```diff
@@ -15,6 +15,8 @@ depends: [
   "portmidi" {>= "0.1"}
   "cmdliner" {>= "1.1.1"}
   "odoc" {with-doc}
+  "portmidi"
+  "cmdliner"
```

Comment:
These are the two lines that still need to be removed.

Reply:
Removed!